src/controllers/item.js

```diff
@@ -1,6 +1,13 @@
+const ItemModel = require('../models/item');
+
 module.exports = {
-  getAllItems: (req, res) => {
-    res.send('All items');
+  getAllItems: async (req, res) => {
+    try {
+      const items = await ItemModel.find();
+      res.json(items);
```

Comment:
When there are no items, `.find()` method doesn't throw an error, instead, it returns an empty array. 
You can check the length of `items` if it is zero or less, then throw an error with message that indicates there are no items

Reply:
Thank you Ammar, good suggestion, I will revise that part.